start-hrwros-nvidia.desktop

```diff
@@ -2,13 +2,18 @@
 Encoding=UTF-8
 Name=HRWROS (Nvidia)
 Comment=Startup the environment for HRWROS for computers with NVIDIA graphics cards.
-Exec=gnome-terminal -e '/bin/bash -c "/bin/mkdir -p /tmp/hrwros_run/dconf && XDG_RUNTIME_DIR=/tmp/hrwros_run /usr/bin/singularity shell -p --nv $(xdg-user-dir DOWNLOAD)/hrwros-09.simg"' --class=hrwros_mooc_class_nv
 Icon=hrwros-mooc-icon-nv.png
+Exec=gnome-terminal -- bash -c "/bin/mkdir -p /tmp/hrwros_run/dconf && XDG_RUNTIME_DIR=/tmp/hrwros_run /
+usr/local/bin/singularity shell -p --nv $(xdg-user-dir DOWNLOAD)/hrwros-09.simg" --class=hrwros_mooc_cla
+ss_nv
```

Comment:
I'm not sure this will work like this (ie: hard-wrapped across multiple lines).

Please revert the wrapping.

Reply:
Same for the other `Exec` entries.

Reply:
ok, it was unintentional